src/sentry/constants.py

```diff
@@ -83,7 +83,7 @@ def get_all_languages():
         'remote', 'get-cli', 'blog', 'welcome', 'features', 'customers', 'integrations', 'signup',
         'pricing', 'subscribe', 'enterprise', 'about', 'jobs', 'thanks', 'guide', 'privacy',
         'security', 'terms', 'from', 'sponsorship', 'for', 'at', 'platforms', 'branding', 'vs',
-        'answers', '_admin', 'support', 'contact', 'onboarding', 'ext', 'extension', 'extensions', 'plugins', 'themonitor',
+        'answers', '_admin', 'support', 'contact', 'onboarding', 'ext', 'extension', 'extensions', 'plugins', 'themonitor', 'settings',
```

Comment:
you can probably bump a few of these down to the next line. also might be worth checking to make sure there isn't a `settings` org if you haven't already?

Reply:
👍 - just checked, no existing `settings` org